config/pitest-suppressions/pitest-filters-suppressions.xml

```diff
@@ -351,15 +351,6 @@
     <lineContent>this.checks = null;</lineContent>
   </mutation>
 
-  <mutation unstable="false">
-    <sourceFile>SuppressionXpathSingleFilter.java</sourceFile>
-    <mutatedClass>com.puppycrawl.tools.checkstyle.filters.SuppressionXpathSingleFilter</mutatedClass>
-    <mutatedMethod>setFiles</mutatedMethod>
-    <mutator>org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator</mutator>
-    <description>Removed assignment to member variable files</description>
-    <lineContent>this.files = null;</lineContent>
-  </mutation>
```

Comment:
@romani this mutation is not possible to cover with unit test as well becuase the mutation is about to remove the assignment so by default it become null. if we remove the assignment then the code becomes non-compilable 

https://kevin222004.github.io/PitestReport/com.puppycrawl.tools.checkstyle.filters/SuppressionXpathSingleFilter.java.html

Reply:
@Kevin222004 , you junit method should be improved.

it shuold set `filter.setFiles("some file");` make sure that xpath is matching.
in same junit and on the same instance of `filter` do
`filter.setFiles(null);` and repeat `accept` and assert on no match of xpath.

with mitation it will match, so junit will fail.

and we need comment over pure junit method to explain why we do non-Input based testing.

Reply:
Ok I have update the test case can you please gave a look to improve it more looks like wrong query is also getting accepted